ceph.spec.in

```diff
@@ -659,6 +659,11 @@ fi
 %{_bindir}/ceph-crush-location
 %{_bindir}/rados
 %{_bindir}/rbd
+%{_bindir}/rbd-replay
+%{_bindir}/rbd-replay-many
+%if 0%{?fedora} || 0%{?rhel} == 6
+%{_bindir}/rbd-replay-prep
```

Comment:
There probably is a compilation conditional fix somewhere to not compile rbd-replay-prep on CentOS 7. Without it we get:
<pre>
Checking for unpackaged file(s): /usr/lib/rpm/check-files /tmp/release/CentOS/WORKDIR/BUILDROOT/ceph-0.87.1-37.gffb76d1.el7.x86_64^M
error: Installed (but unpackaged) file(s) found:^M
   /usr/share/man/man8/rbd-replay-prep.8.gz^M
^M
^M
RPM build errors:^M
    Installed (but unpackaged) file(s) found:^M
   /usr/share/man/man8/rbd-replay-prep.8.gz^M
</pre>